src/linker/pipeline.py

```diff
@@ -44,12 +50,5 @@ def _get_implementations(self) -> Tuple[Implementation, ...]:
         implementations = []
         for step in self.steps:
             implementation_name = self.config.pipeline["steps"][step.name]["implementation"]
-            implementation_dir = (
-                Path(os.path.realpath(__file__)).parent
-                / "steps"
-                / step.name
-                / "implementations"
-                / implementation_name
-            )
-            implementations.append(Implementation(step, implementation_dir))
+            implementations.append(Implementation(step.name, implementation_name))
```

Comment:
Maybe catch and batch errors thrown during Implementation and Pipeline validations